etl/steps/data/garden/gapminder/2023-09-18/under_five_mortality.excluded_countries.json

```diff
@@ -0,0 +1,2 @@
+[
```

Comment:
It'd be better to remove the file (and its mention in the code) instead of leaving it empty.